packages/editor-ui/src/components/Error/NodeErrorView.vue

```diff
@@ -696,9 +702,14 @@ async function onAskAssistantClick() {
 		}
 	}
 
-	&__assistant-button {
+	&__button {
 		margin-left: var(--spacing-s);
 		margin-bottom: var(--spacing-xs);
+		flex-direction: row-reverse;
```

Comment:
I guess this is ok for now if this is the only instance where we need show an icon on the left. If we end in more situations like this I think it's better to update the `Button` component so it has slots for icons on both sides.